assginment7/src/Main.java

```diff
@@ -0,0 +1,17 @@
+import dao.StudentDao;
+import models.Student;
+
+import java.util.ArrayList;
+
+public class Main {
+
+    public static void main(String[] args) {
+        StudentDao studentDao = new StudentDao();
+        studentDao.createConnectionAndStatement();
```

Comment:
You can call this method into each CRUD operation method internally so that Connection and Statement 
will be created as and when a method requires to create one.